src/main/java/net/openhft/chronicle/wire/VanillaMethodReaderBuilder.java

```diff
@@ -138,10 +139,12 @@ private MethodReader createGeneratedInstance(Object... impls) {
                 }
             }
         } catch (Throwable e) {
+            if (Jvm.getBoolean(DISABLE_PROXY_REFLECTION, true))
+                throw Jvm.rethrow(e);
             classCache.put(fullClassName, COMPILE_FAILED);
             Jvm.warn().on(getClass(), "Failed to compile generated method reader - " +
```

Comment:
this code is not exercised any more. We should probably add in a test to exercise it